### ISSUE TITLE
irmin-pack: Activity log

### ISSUE THREAD

**Issue by Ngoguey42** [mirage/irmin#1856]
In order to simplify debugging we could add an append only ascii file that lists the timestamps of the creation, openings, gc starts and gc ends.

**Comment by Ngoguey42** [comment]
How bad would it be that open_ro writes something in such a file?

**Comment by tomjridge** [comment]
I agree that it might be very useful to have GC write its own internal logs to a separate file (whilst also reporting logs to the usual places). 

Since it's a log file, open_ro could write things in this file (?not sure why we wouldn't want it to if needed)
